build-gradle-plugin/build.gradle

```diff
@@ -1,30 +1,23 @@
 plugins {
     id("org.gradle.kotlin.kotlin-dsl") version "1.4.2"
     id("java")
-    id("groovy")
     id("kotlin")
     id("java-gradle-plugin")
     id("maven-publish")
 }
 
 dependencies {
-    implementation(gradleApi())
-    implementation(localGroovy())
     implementation("com.android.tools.build:gradle:4.1.0")
     implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version")
     implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version")
 }
 
 ext {
-    bintrayName = 'build-gradle-plugin'
-    publishedArtifactId = 'build-gradle-plugin'
-
-    libraryName = 'Ackee Gradle Plugin'
-    libraryDescription = 'Adds several extensions and tasks used in all our projects'
+    PUBLISH_ARTIFACT_ID = 'build-gradle-plugin'
+    POM_DESCRIPTION = 'Gradle plugin with some useful configuration shared among projects'
 }
 
-apply from: rootProject.file('gradle/bintray/install.gradle')
-apply from: rootProject.file('gradle/bintray/bintray.gradle')
+apply from: "${rootProject.projectDir}/gradle/mavencentral/publish.gradle"
 
 // This allows you to publish to local maven repository on your machine via publishToMavenLocal task
```

Comment:
this can be removed, right?

Reply:
d956d576a63e6440578d5cab79a663f352f02ccc